slacker/utility.py

```diff
@@ -72,3 +74,26 @@ def parse_args():
 
   args = parser.parse_args(args)
   return (args, cmd_args)
+
+def verify_token(token):
+  try:
+    return SlackAPI(token).post('auth.test')
+  except Exception as ex:
+    create_logger(__name__).warning(ex)
+  return None
+
+def workspace_prompt(msg = 'Input workspace token: '):
```

Comment:
Can we rename this to be `workspace_token_prompt` so it's a bit more informative. At first I wasn't sure what it was doing.

Reply:
Sure, I can rename it.